### ISSUE TITLE
[Feature request] Translating Language Identifiers

### ISSUE THREAD

**Issue by caramelomartins** [keeweb/keeweb#374]
Hello,

I have a suggestion I would like to discuss. Shouldn't we be converting language names also? 

When translating for Portuguese, we'd have:
```
English -> Inglês
Portuguese -> Português
```

And so on.

What do you think?

**Comment by antelle** [comment]
This is by design, because if a user switches language accidentally, English must be English, not 'イングリッシュ'. 
Additionally, we can translate names and show the translation in brackets (e.g. `English (イングリッシュ)`), but:
1. it will not make any sense (what's the point to see other language names translated, if everyone need either own language, or English?)
2. N<sup>2</sup> strings will be required where N is the number of languages
3. adding a new language will mean, we now need translate its name to all locales

Basically, I don't see any point translating them.

**Comment by caramelomartins** [comment]
@antelle I undestand your point. I was thinking from a UI perspective in which a user might not understand English but even if it one can't it will be difficult to chance languages in the first place so...it would add a lot of overhead without any practical gain. I understand. 

Thanks for replying anyway!

**Comment by antelle** [comment]
If a user doesn't understand English, translating language names won't help here, because by default we show English ui, and you have to switch it first.
In next release, wel'll select the locale based on user preference, so it will solve this, to some extent.

**Comment by caramelomartins** [comment]
@antelle I was agreeing with your choice :+1:
